libraries/Microsoft.Bot.Builder.Ai.Translation/TranslationMiddleware.cs

```diff
@@ -120,7 +120,7 @@ public virtual async Task OnTurnAsync(ITurnContext context, NextDelegate next, C
                         }
                         else
                         {
-                            sourceLanguage = await _languageStateProperty.GetAsync(context).ConfigureAwait(false) ?? "en";
+                            sourceLanguage = await _languageStateProperty.GetAsync(context, () => "en").ConfigureAwait(false) ?? "en";

```

Comment:
> "en" [](start = 97, length = 5)

Same here, move this into property on middleware that can be set in the constructor